packages/visx-demo/src/sandboxes/visx-xychart/ExampleControls.tsx

```diff
@@ -42,14 +43,20 @@ type ProvidedProps = {
     y: Accessors;
     date: Accessor;
   };
+  animationTrajectory: AnimationTrajectory;
+  annotationDataKey: keyof Accessors | null;
```

Comment:
in this file, just added the annotation state + re-ordered several controls to group them more logically (lots of controls were added over time so it's a little overwhelming)